dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -29,9 +30,11 @@ public class ObjectCreationNode extends Node {
     /** The enclosing expression of the object creation or null. */
     protected final @Nullable Node enclosingExpression;
 
-    // TODO: See issue 376
-    /** The constructor node of the object creation. */
-    protected final Node constructor;
+    /**
+     * The identifier node of the object creation. A generic constructor identifier can refer to the
+     * ParameterizedTypeNode, while a non-generic constructor can refer to the ClassNameNode.
```

Comment:
I would probably first list the non-generic case and then second the generic case. The `to the` is a bit odd - maybe `to a`?